features/0510-dif-pres-exch-attach/README.md

```diff
@@ -241,7 +314,158 @@ Complete message example:
     }]
 }
 ```
+</details>
+
+
+### Supported Features of Presentation-Exchange
+
+Level of support for Presentation-Exchange features:
+
+Feature|Notes
+-------|-----
+`presentation_definition.input_descriptors.id`|
+`presentation_definition.input_descriptors.name`|
+`presentation_definition.input_descriptors.purpose`|
+`presentation_definition.input_descriptors.schema.uri`|URI for the credential's schema.
+`presentation_definition.input_descriptors.constraints.fields.path`|Array of JSONPath string expressions as defined in [section 8](https://identity.foundation/presentation-exchange/#jsonpath-syntax-definition). REQUIRED as per the spec.
+`presentation_definition.input_descriptors.constraints.fields.filter`|JSONSchema descriptor.
+`presentation_definition.input_descriptors.constraints.limit_disclosure`|`preferred` or `required` as defined in [the spec](https://identity.foundation/presentation-exchange/#input-descriptor-object) and as supported by the proof mechanisms processed by the Holder and the Verifier.
+`presentation_definition.format`|For JSONLD-based credentials: `ldp_vc` and `ldp_vp`.
+`presentation_definition.format.proof_type`|For JSONLD-based credentials: `Ed25519Signature2018` and `BbsBlsSignatureProof2020`.
```

Comment:
Note the proof type of the presentation can differ from the proof type of the credential.

Also `BbsBlsSignatureProof2020` is a proof type for the derived credential.